test/unit/deployment.bats

```diff
@@ -700,3 +700,34 @@ load _helpers
    actual=$(echo "$object" | yq '.label2'| tee /dev/stderr)
    [ "${actual}" = 'value2' ]
 }
+
+#--------------------------------------------------------------------
+# extraArgs
+
+@test "controller/Deployment: extraArgs not set by default" {
+  cd `chart_dir`
+  local object=$(helm template \
+      -s templates/deployment.yaml  \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.containers[1].args | select(documentIndex == 1)' | tee /dev/stderr)
```

Comment:
Something like this might make these more robust:

```suggestion
      yq 'select(.kind == "Deployment") | .spec.template.spec.containers[] | select(.name == "manager") | .args' | tee /dev/stderr)
```

Reply:
+1, I think I was following what we have in the `extraLabels` tests.

Reply:
Updated in https://github.com/hashicorp/vault-secrets-operator/pull/376/commits/a6580774a37a1cd0178c88d9746c8dbf829ea4f6